modules/agar/src/main/ts/ephox/agar/touch/Touches.ts

```diff
@@ -17,6 +17,7 @@ const point = (type: string, element: Element<any>, x: number, y: number): void
 
   // Adapted from https://stackoverflow.com/a/42447620/11275515
   if (typeof TouchEvent === 'function' && typeof Touch === 'function') {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
is there an `es-ignore` we could use instead?

Reply:
Unfortunately, no as typescript won't compile without it. The core issue is because our `dom-global` types are quite old/incorrect so it thinks `Touch` should take 0 arguments.

Reply:
I didn't mean change the code, just is there a general "es ignore" instead of using an ES disable to allow a TS ignore

Reply:
We should probably turn off the @typescript-eslint/ban-ts-ignore rule. It seems quite silly to have a lint to ban something that we'll only use when absolutely needed...

Reply:
I have disabled @typescript-eslint/ban-ts-ignore again and added it to the eslint plugin - I'll wait until I've gone through the rest of the tinymce lints to see if I need to make more modifications to the eslint plugin before I publish a new version.